src/twisted/internet/test/test_inlinecb.py

```diff
@@ -146,18 +148,43 @@ def calling():
         d = calling()
         f = self.failureResultOf(d)
         tb = f.getTraceback()
-        self.assertIn("in erroring", tb)
+        self.assertIn("erroring", tb)
```

Comment:
can we still have a more specific assertion here?

I am not sure why we need this change for older python versions.

Reply:
This change was in error, I was looking at the wrong test.